pages/listings/listing/student-application-viewed.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
```

Comment:
is this not ment to be application team work ? @Jenfer93

Reply:
I think, and hope the name is just wrong, have to investigate what has happend!